sci-chemistry/vmd/vmd-1.9.3.ebuild

```diff
@@ -0,0 +1,248 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cuda eutils flag-o-matic multilib prefix python-single-r1 toolchain-funcs
+
+DESCRIPTION="Visual Molecular Dynamics"
+HOMEPAGE="http://www.ks.uiuc.edu/Research/vmd/"
+SRC_URI="
+	${P}-gentoo-patches.tar.xz
+	${P}.src.tar.gz"
```

Comment:
Btw, `http://www.ks.uiuc.edu/Research/vmd/${P}/files/final/${P}.src.tar.gz` should just work.